controllers/deleteCommentByIdController.js

```diff
@@ -0,0 +1,10 @@
+const { removeCommentById } = require("../models/comments");
+
+exports.deleteCommentById = (req, res, next) => {
+  const { comment_id } = req.params;
+  removeCommentById(comment_id)
+    .then((deleted) => {
+      res.status(204).send({ deleted });
```

Comment:
express will not send a response body with a status of 204 - because a 204 means 'no content'

for this endpoint, it's totally fine to not send a response body, we know the deletion has worked because the 204 comes back.
`res.status(204).send();` is a totally valid way to send our response for this endpoint